app/assets/stylesheets/core/_settings.scss

```diff
@@ -1 +1,3 @@
 @import "foundation_and_overrides";
+@import "foundation";
+@include foundation-everything;
```

Comment:
Why we use different file name style for `foundation_and_overrides` and `foundation-everything` ?

Do we need to use double quotes here as well?